rustls/tests/api.rs

```diff
@@ -2936,6 +2936,64 @@ fn sni_resolver_rejects_bad_certs() {
     );
 }
 
+/// Represents a SigningKey that returns None for its SPKI.
```

Comment:
nit: maybe worth calling out the default impl? 

```suggestion
/// Represents a SigningKey that returns None for its SPKI via the default impl
```